docs/build/guides/conventions/stellar-asset-contract.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Integrate Stellar Assets Contracts
+description: Test and use Stellar assets in a Soroban smart contract
+---
+
+From the contract perspective Stellar Asset Contract is not different from any other token that implements the Soroban [SEP-41 Token Interface]. The Rust SDK contains a pregenerated client for any contract that implements the token interface:
+
+```rust
+use soroban_sdk::token;
+
+struct MyContract;
+
+#[contractimpl]
+impl MyContract {
+  fn token_fn(e: Env, id: Address) {
```

Comment:
4-space indentations in the Rust code examples, please. If you run `yarn format:mdx` in your local repo it "should" fix that for you (assuming you're editing locally, and have installed all the dependencies, that is).

Reply:
Thanks for fixing this manually. There may be something weird happening with our formatter (probably a weird thing with the v3 upgrade). I'll do some digging 🙏🏻